src/compileApplicationToWasm.js

```diff
@@ -117,16 +117,16 @@ export async function compileApplicationToWasm(
 
   try {
     let wizerProcess = spawnSync(
-      wizer,
+      `"${wizer}"`,
       [
         "--inherit-env=true",
         "--allow-wasi",
         "--dir=.",
         ...starlingMonkey ? [`--dir=${dirname(wizerInput)}`] : [],
         `--wasm-bulk-memory=true`,
         "-r _start=wizer.resume",
-        `-o=${output}`,
-        wasmEngine,
+        `-o="${output}"`,
+        `"${wasmEngine}"`,
```

Comment:
Normally `spawnSync` doesn't require escaping of arguments as it automatically handles this (and this is also why `windowsVerbatimArguments` is an option.

Therefore unless we can prove there is a test case where this fails, I'm pretty sure this code path should support spaces okay in the command and arguments.

Reply:
Unfortunately it still needs the double quotes around the input on L#120 and around the args on L#128-L#129